lib/install/dnsmasq.rb

```diff
@@ -0,0 +1,38 @@
+require_relative "../doctor/dnsmasq"
+
+module Install
+  class Dnsmasq
+    def call
+      install_dnsmasq
+      configure_etc_resolver_devgovuk
+      configure_usr_local_etc_dnsmasq
+      configure_usr_local_etc_dnsmasq_developmentconf
+      restart_dnsmasq
+      verify_dns
+      puts "✅ Dnsmasq installation and configuration complete!"
+    end
+
+  private
+
+    def install_dnsmasq
+      return if Doctor::Dnsmasq.new.installed?
+      puts "⏳ Installing dnsmasq"
+      system("brew install dnsmasq")
```

Comment:
I wonder if we should have something like `system("brew install dnsmasq") if %x[uname -a].include?("Darwin")` just in case someone's not on a Mac?

Reply:
Yeah, I'm thinking actually the location of the config files only apply when installed by brew so I think I'll put a check right at the top which asks the user to confirm if they really want to do this if they're not on a Mac.